libs/applePayments.js

```diff
@@ -0,0 +1,112 @@
+const nconf = require('nconf');
+const iap = require('in-app-purchase');
+const request = require('request');
+const USERS_BATCH = 10;
+const BASE_URL = nconf.get('BASE_URL');
+const subscriptions = require('../libs/subscriptions');
```

Comment:
USER_BATCH and BASE_URL should be separated by 1 line from the `require` statements, for clarity